mysqld_exporter.go

```diff
@@ -52,6 +63,56 @@ func NewMySQLExporter(dsn string) *Exporter {
 			Help:      "The last scrape error status.",
 		}),
 		metrics: map[string]prometheus.Gauge{},
+		commands: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "commands_total",
+			Help:      "Number of executed mysql commands.",
+		}, []string{"command"}),
+		connection_errors: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "connection_errors_total",
```

Comment:
I don't see this documented on https://dev.mysql.com/doc/refman/5.1/en/server-status-variables.html